packages/react-components/stories/progressbar.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { ProgressBar as ProgressBar_ } from '../lib/progressbar';
+import { LinearProgressProps } from '@material-ui/core';
+
+export default {
+  title: 'Progress Bar',
```

Comment:
Move this to a `Robots` section if `ProgressBar` and `CircularProgressBar` is moved to be next to `RobotInfo`.